generated/tests/fixtures.py

```diff
@@ -0,0 +1,151 @@
+import pytest
```

Comment:
I don't think we should copy the `tests/` directory to `generated/tests/`. We're not going to codegen tests.

Reply:
Agree, let's keep those separate. In fact, I really dislike that the `tests/` folder is under `nidaqmx/` at the moment. So if you wanted to move it to the root now, I wouldn't mind.

Reply:
I have moved `tests/` to the root folder and currently the test files are still referencing the `nidaqmx` module present externally and not the generated files. Once the code generation is complete we can take up the task updating the tests to make use of the generated files.

Reply:
I think changing the tests to reference the generated files is unnecessary. Once we update `pyproject.toml` to use the generated files, then `poetry install` will install the generated files as the `nidaqmx` module, and it will work just like when you install it from PyPI.